recipe/patches/0001-remove-upper-bounds.patch

```diff
@@ -0,0 +1,27 @@
+From 88b437f4af344473ba0dcc5c83afbe9bce482fb1 Mon Sep 17 00:00:00 2001
+From: Udo-Peter Steyer <ous8292@gmail.com>
+Date: Thu, 19 Sep 2024 12:20:49 -0500
+Subject: [PATCH] remove upper bounds
+
```

Comment:
why do you remove the upper bounds?

Reply:
I added a comment saying we followed the conda-forge recipe. Note that the versions of the two packages we have built are higher than the required pinnings